auction9-frontend/src/app/home/home.component.ts

```diff
@@ -10,14 +11,24 @@ export class HomeComponent implements OnInit {
   // array of all active auctions to be sent to data-table component
   tableData: any[];
   tableHeaders: string[];
+  isLoggedIn: boolean;
 
   constructor(private auctionService: AuctionService) { }
 
   ngOnInit(): void {
-    this.auctionService.getActiveAuctions().then((data: []) => {
-     this.tableData = Array.from(data);
-     this.tableHeaders = ['auctionID', 'title', 'price', 'info'];
-   });
-  }
 
+    Auth.currentAuthenticatedUser()
+      .then((user) => {
+        this.auctionService.getActiveAuctions().then((data: []) => {
```

Comment:
We have almost the same code in success and fail callback, should we consider to make it in some other way, or use `info` column no matter if user is logged in.